plugins/modules/helm_template.py

```diff
@@ -99,6 +99,12 @@
     type: list
     elements: str
     version_added: 2.4.0
+  validate:
+    description:
+      - Validate your manifests against the Kubernetes cluster you are currently pointing at. This is the same validation performed on an install.
+    required: false
+    type: bool
+    default: false
```

Comment:
```suggestion
    default: false
    version_added: 2.5.0
```

Reply:
forced pushed a changelog fragment and added `version_added`